src/client/api.ts

```diff
@@ -32,7 +33,13 @@ export interface IExtensionApi {
      * @memberof IExtensionApi
      */
     ready: Promise<void>;
+    /**
+     * Do not use this to monitor execution state of cells of Native Notebooks (use VS Code API).
+     */
     readonly onKernelStateChange: Event<KernelStateEventArgs>;
+    /**
+     * Do not use this to register Cell Toolbar icons for Native Notebook.
+     */
     registerCellToolbarButton(
```

Comment:
This is needed for the Interactive window

Reply:
Yes, I agree, i'm only mentioning what this should not be used for.